x-pack/plugin/graph/src/main/java/org/elasticsearch/xpack/graph/rest/action/RestGraphAction.java

```diff
@@ -63,22 +57,10 @@
     public static final ParseField TERM_FIELD = new ParseField("term");
 
     public RestGraphAction(RestController controller) {
-        // TODO: remove deprecated endpoint in 8.0.0
-        controller.registerWithDeprecatedHandler(
-                GET, "/{index}/_graph/explore", this,
-                GET, "/{index}/_xpack/graph/_explore", deprecationLogger);
-        // TODO: remove deprecated endpoint in 8.0.0
-        controller.registerWithDeprecatedHandler(
-                POST, "/{index}/_graph/explore", this,
-                POST, "/{index}/_xpack/graph/_explore", deprecationLogger);
-        // TODO: remove deprecated endpoint in 8.0.0
-        controller.registerWithDeprecatedHandler(
-                GET, "/{index}/{type}/_graph/explore", this,
-                GET, "/{index}/{type}/_xpack/graph/_explore", deprecationLogger);
-        // TODO: remove deprecated endpoint in 8.0.0
-        controller.registerWithDeprecatedHandler(
-                POST, "/{index}/{type}/_graph/explore", this,
-                POST, "/{index}/{type}/_xpack/graph/_explore", deprecationLogger);
+        controller.registerHandler(GET, "/{index}/_graph/explore", this);
```

Comment:
It looks like we removed the deprecated `_xpack` endpoints here instead of the ones containing `{type}`. We could remove the ones with `{type}` in this PR, and leave the `_xpack` removal for a future PR?

Reply:
Oops, good catch - will fix.